pytest_splunk_addon/helmut/manager/confs/rest/conf.py

```diff
@@ -6,20 +6,21 @@
 from future import standard_library
```

Comment:
Does pyupgrade not remove future lib references?

Reply:
No, unfortunately. We will need to remove it manually in the next PRs.